uber/models.py

```diff
@@ -525,7 +525,7 @@ def unset_volunteering(self):
 
     @property
     def badge_cost(self):
```

Comment:
Derp.  I forgot that ``_local`` properties raise an exception (as opposed to returning ``None``) if the underlying ``datetime`` is ``None``.  Which... probably is a bad design decision?  That doesn't seem like a feature, but it's a bigger change than I felt comfortable making now.